src/main/java/seedu/duke/parser/CommandExecutor.java

```diff
@@ -362,7 +363,7 @@ private static void executeDeleteTrip(int tripIndex) {
      * @param attributeToEdit String of all attributes to be added and their new values
      */
     private static void editTripPerAttribute(Trip tripToEdit, String attributeToEdit) throws ForceCancelException {
-        String[] splitCommandAndData = attributeToEdit.split(" ");
+        String[] splitCommandAndData = attributeToEdit.split(" ", 2);
```

Comment:
You might want to consider removing magic numbers here

Reply:
thanks for the spot! 😎